owncast_for_windows.md

```diff
@@ -52,20 +53,40 @@ Here is the list for all the prerequisites required ->
  - [ffmpeg](https://ffmpeg.org/download.html)
  - Install the [Go toolchain](https://golang.org/dl/) (1.18 or above).
 
-### Step 4: Installing Owncast Backend
+### Follow Step 4 if you want to install and use Owncast.
+
+### Step 4: Installing Owncast for streaming.
+One of the many ways to install Owncast is to use the Quick Installer script.
+In the WSL2 terminal run, the following command
+
+```
+curl -s https://owncast.online/install.sh | bash  
+cd owncast  
+./owncast   
+```
+
+If everything is installed correctly you will be able to access the admin interface at port `localhost:8080/admin`.  
+Here are other ways listed you can use to install Owncast -[Owncast Quickstart](https://owncast.online/quickstart/)
+
+
+### Follow Steps 5 and 6 for installing and building Owncast for development.
+
+### Step 5: Installing Owncast Backend
 The Owncast backend is a service written in Go.
 1. Clone the repo. `git clone https://github.com/owncast/owncast`
 1. `go run main.go` will run from the source.
 1. Visit `http://yourserver:8080` to access the web interface or `http://yourserver:8080/admin` to access the admin.
 1. Point your [broadcasting software](https://owncast.online/docs/broadcasting/) at your new server and start streaming.
 
-### Step 4: Installing Owncast Frontend
+### Step 6: Installing Owncast Frontend
```

Comment:
Sorry to be picky, but these steps as stated are a bit confusing. It says you have to 1) Install Owncast 2) Setup the backend 3) Setup the frontend.

Maybe _using_ owncast and _developing_ owncast should be split up so it doesn't look like you have to install clone the repo and run the backend and frontend in order to run the software.